offchain-modules/package.json

```diff
@@ -121,9 +130,14 @@
     ]
   },
   "lint-staged": {
-    "packages/*/src/**/*.ts": [
+    "src/**/*.ts": [
```

Comment:
this part and the husky part seems like a merge error.
should use the version in main branch.